Design/Vehicle-Sizing/Pressure Drop Calculator.py

```diff
@@ -0,0 +1,127 @@
+    
+    gravity = 32.17405 #ft/s^2 - https://www.engineeringtoolbox.com/mass-weight-d_589.html
+
+#Based on massflow input, calculate the velocity. Assumed large tank, so only velocity considered is that at point 2
+#Assume massflow used by engine is a constant value along all points outside the tank
+    velocity = massflow / (density*area)
+
+    reynolds = velocity*inner_diameter/(kv) #https://www.engineeringtoolbox.com/reynolds-number-d_237.html - circular pipe
+
+#friction factor calculation for laminar flow
+#https://www.nuclear-power.net/nuclear-engineering/fluid-dynamics/major-head-loss-friction-loss/friction-factor-for-laminar-flow/
+    if reynolds < 2000:
+        f = 64/reynolds
+    
+#friction factor calculation for turbulent flow with Re < 100,000
+#https://www.nuclear-power.net/nuclear-engineering/fluid-dynamics/major-head-loss-friction-loss/friction-factor-turbulent-flow-colebrook/
+    if reynolds > 2000:
+        def friction(f):
+            return (-2*math.log10(2.51/(reynolds*math.sqrt(f))+((pipe_roughness/inner_diameter)/3.72)))**2 - 1/f
+
+        f = fsolve(friction,0.01)
+
+#####################################################################################################################
+
+#Can't find another source that uses this equation for such a relatively low Reynold's number, uncomment if verified
+
+##friction factor calculation for turbulent flow with Re > 100,000    
+##https://www.pipeflowcalculations.com/pipe-valve-fitting-flow/flow-in-pipes.php#Bernoulli-theorem
+#    if reynolds > 100000:
+#        f = (2.0*math.log10(inner_diameter/pipe_roughness)+1.14)**(-2)
+
+#####################################################################################################################    
+    
+    density_calc = density*0.031081*(12**3) #lb/in^3 -> slug/ft^3
+    pipe_length_calc = pipe_length/12 #in -> ft
+    velocity_calc = velocity/12 #in/s -> ft/s
+    inner_diameter_calc = inner_diameter/12 #in -> ft
+    
+    head_loss_major = (f*pipe_length_calc*velocity_calc**2)/(2*inner_diameter_calc*gravity)
+    head_loss_minor = (k_factor*velocity_calc**2)/(2*gravity)
+    
+    total_head_loss = head_loss_major + head_loss_minor
+    
+    return reynolds, f, total_head_loss, velocity
+
+def pressure_drop_calc(length,pressure_initial,density,velocity,head_loss): 
+    gravity = 32.17405
+    density_calc = density*0.031081*(12**3) #lb/in^3 -> slug/ft^3
+    length_calc = length/12 #in -> ft
+    velocity_calc = velocity/12 #in/s -> ft/s
+    
+    pressure_change = head_loss * density_calc * gravity - density_calc * length_calc * gravity + 1/2 * velocity_calc**2 * density_calc
+    psi_change = pressure_change/144
+    
+    return psi_change
+
+def cv_calc(massflow,sg,delP,density):
+    Q = massflow/density #in^3/s
+    vol_flow = 0.004329 * Q *60 #gal/min
+    Cv = vol_flow * math.sqrt(sg)/math.sqrt(delP)
+    
+    return Cv
+
+pressure = 1000 #psi
```

Comment:
All of these individual statements should be wrapped in a function. Perhaps use a config file input like openrocketengine?